test/blackbox-tests/test-cases/inline_tests/run.t

```diff
@@ -1,6 +1,6 @@
   $ env -u OCAMLRUNPARAM jbuilder runtest simple
            run alias simple/runtest (exit 2)
-  (cd _build/default/simple && ./.foo_simple.inline-tests/run.exe)
+  (cd _build/default/simple && _build/default/simple/.foo_simple.inline-tests/run.exe)
```

Comment:
This looks suspicious. The rest of the PR looks correct to me

Reply:
Is this an issue with `Path.reach_for_running`? Im' simply calling it here like this:

```
Path.(relative ~from:(of_string "_build/default/simple") (of_string _build/default/simple/.foo_simple.inline-tests/run.exe")
```